doc-Provisioning_Virtual_Machines_and_Hosts/topics/appe-creating-a-service-for-provisioning-vm.adoc

```diff
@@ -0,0 +1,169 @@
+
+The following three items are required to make a service available to users from the {product-title_short} service user interface:
+
+. *Service Dialog*: A service dialog is where you allow users to configure virtual machine or instance options. A service dialog determines which options users get to change. You can choose the options presented to the user. You could just give them the option to set the service name, or you could have them change all of the provisioning dialog options. 
+
+. *Service Catalog*: A service catalog is used to group service dialogs together.
+
+. *Service Catalog Item*: A service catalog item is the actual service which combines a service dialog with a provisioning dialog.
+
+[[provisioning-dialog]]
+=== Examining the Provisioning Dialog
+
+Understanding provisioning dialogs is important to work with services in {product-title_short}. Every provider in {product-title_short} comes with a sample provisioning dialog covering the options specific to that provider. A provisioning dialog presents the basic configuration options for a virtual machine or instance. When you begin the process of provisioning a virtual machine or instance via {product-title_short}, you are presented with a provisioning dialog where you set certain options for the virtual machine or instance. The options presented are dependent on the provider you are using. For instance, a cloud provider might have flavors of instances, whereas an infrastructure provider might allow you to set the memory size or number of CPUs on a virtual machine.
+
+To see the sample provisioning dialog for Red Hat Virtualization: 
+
+. Navigate to menu:Automation[Automate > Customization].
+. Expand the *Provisioning Dialogs* accordion and select *Sample RedHat VM Provisioning Dialog* under *VM Provision*. 
+
+This is a textual representation of the dialog you will get when you provision a virtual machine.
+
+[[create-service-dialog]]
+=== Creating a Service Dialog
+
+Using a service dialog you can determine which options users get to change. When users access a service, the majority of options available to them are preset and cannot be altered. A service dialog exposes some of those options to the user so that even if they are ordering a basic RHEL 7 machine, for example, they can at least choose the amount of memory, virtual CPUs, or other options available to the instance they order. In cases where certain fields must be unique, such as the name of virtual machines in Red Hat Virtualization, users must enter their own unique name for the virtual machine they choose or the operation will fail, so this field must be exposed.
+
+[NOTE]
+====
+You cannot create a service without assigning it a service dialog, so even if you do not want to expose any details to users, you must create at least a simple service dialog with a *Submit* and *Cancel* button.
```

Comment:
Here I would recommend breaking this out into more than one sentence. Maybe something like "A service requires an assigned service dialog, regardless of exposing details to user. A simple service dialog with a 'Submit' and 'Cancel' button is required at minimum."

Reply:
Fixed.